src/Engine/GameEvents/CommanderAbilityEvent.java

```diff
@@ -18,13 +19,23 @@ public CommanderAbilityEvent(CommanderAbility ability)
   @Override
   public GameAnimation getEventAnimation(MapView mapView)
   {
-    return null;
+    // TODO: CO Ability intro splash
+    GameAnimation anim = null;
+    if( !gameEvents.isEmpty() )
+    {
+      // Just grab the first one for now; TODO create compound animations.
+      anim = gameEvents.peek().getEventAnimation(mapView);
+    }
+    return anim;
   }
```

Comment:
Super random idea for setting up ability buff animations:
The animation callsite takes in a `UnitModel` with zero'd stats + the "delta", and maybe a "is this a permanent buff" flag. (TBD whether it's a set of units or a single unit per animation)
That would still leave out stuff like healing or refresh, but it would make for a "reasonable" interface that automatically stays current with all available knobs to turn and allows for multiple changes at once.